syncserver/Cargo.toml

```diff
@@ -42,10 +41,6 @@ hostname = "0.3.1"
 hawk = "5.0"
 hmac = "0.12"
 mime = "0.3"
-reqwest = { workspace = true, features = [
```

Comment:
Looks like both reqwest and `env_logger` weren't being used by this crate, so I removed them while cleaning this up